### ISSUE TITLE
Update for v0.10

### ISSUE THREAD

**Issue by gjjeffers** [dotnet/docs#10390]
This line is broken in v0.10 as the constructors have been made internal:
var trainer = new SdcaMultiClassTrainer(_mlContext,DefaultColumnNames.Label, DefaultColumnNames.Features);

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 77fc5686-2288-d607-8136-3146ed02dbf6
* Version Independent ID: 8a16b9e9-b5cb-a55e-2135-191bc62c1656
* Content: [Use ML.NET in a GitHub issue multiclass classification scenario](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/github-issue-classification)
* Content Source: [docs/machine-learning/tutorials/github-issue-classification.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/github-issue-classification.md)
* Product: **dotnet-ml**
* GitHub Login: @JRAlexander
* Microsoft Alias: **johalex**

**Comment by JRAlexander** [comment]
Thanks, @gjjeffers! I'm updating this tutorial currently and hope to have the PR out later tonight. I'll reference it here. Sorry for the inconvenience!

**Comment by mminertrinity** [comment]
Just curious, has this issue been resolved? I'm running into the same problem, but totally understand it may be taking time to address. Thanks @JRAlexander!

**Comment by mminertrinity** [comment]
@JRAlexander thanks for addressing this, but I still seem to be running into the same issue. I checked your commit in #10419 and only saw adjustments to the sentiment exercise. Any chance you could point me in the right direction?

**Comment by JRAlexander** [comment]
Sorry,@mminertrinity, added this to the wrong PR. The updated tutorial topic should be completed tomorrow. I've updated the code sample here: https://github.com/dotnet/samples/pull/650. It should have everything you need for the code.  Again, apologies for the confusion and the delay!

**Comment by mminertrinity** [comment]
Thanks so much @JRAlexander! I was able to restructure a fair amount of the code using the iris example, but this is definitely helpful to check my materials. Thanks again!

**Comment by JRAlexander** [comment]
You are very welcome, @mminertrinity! I'm putting the finishing touches on the tutorial topic now.
